### PR TITLE
Added Nvidia Control Panel Settings

### PR DESCRIPTION
**Pull request by Sirbucket**
A great guide on Nvidia control panel settings added, many of the settings are shared with Radeon so the guide can be used for AMD GPUs as well.

### CODE REVIEW

File: docs/os/windows.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Use [TCP Optimizer](https://www.speedguide.net/downloads.php) to optimize your i
 
 Big idea: disable as many overlays as possible. They interrupt the rendering process with their own, which can severely affect performance in many causes. Common examples of overlays are: Discord and Nahimic.
 
-### Disabling Nahimic
+## Disabling Nahimic
```
Comment (mastercoms) [inline docs/os/windows.md:67]: Nahimic is an overlay, so it should be listed under disable overlays.

File: docs/os/windows.md [hunk 2]
```diff
@@ -86,3 +86,9 @@ Badly programmed kernel-mode device drivers might cause latency issues, which le
 [Enabling Message Signaled Interrupts (PCIe MSI)](https://forums.guru3d.com/threads/windows-line-based-vs-message-signaled-based-interrupts-msi-tool.378044/) for all drivers is a great way to lower DPC latency caused by drivers (ring0). If you have an NVIDIA card, you can use [NVCleanstall](https://www.techpowerup.com/download/techpowerup-nvcleanstall/) to enable this, with the added benefit of fully customizing your driver install.
 
 Use [LatencyMon](https://www.resplendence.com/latencymon) and [DPC Latency Checker](https://www.thesycon.de/eng/latency_check.shtml) to analyze latency issues caused by kernel-mode device drivers. If they report issues, try updating your drivers or installing alternate drivers.
+
+## Nvidia Control Panel Settings (Can be translated to AMD Radeon Settings)
+
+[Nvidia Control Panel Optimization](https://www.youtube.com/watch?v=ugadZ86uTik) Does an excellent job explaining how to configure the Nvidia control panel and goes over each setting in depth and why to use it/not use it. Many settings in the Nvidia control panel are shared with the Radeon panel so you can go off of this guide for it as well.
```
Comment (mastercoms) [inline docs/os/windows.md]: This is meant to be a text guide, with additional sources for users to expand on the reasoning behind a recommendation or to better understand it. It shouldn't be a way to list a bunch of other tutorials, especially YouTube videos.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/os/windows.md (hunk 2 above)
Comment (Sirbucket) [reply docs/os/windows.md]: Ahh understood, did you see my one on power though (It has a tutorial video but I can remove that one if you don't want and I can summarize all the settings into text for this part.)

File: docs/os/windows.md (hunk 1 above)
Comment (Sirbucket) [reply docs/os/windows.md:67]: I did that so it fit in uniform with the other ones, all the others had 2 ## so I thought 3 looked weird.

File: docs/os/windows.md (hunk 2 above)
Comment (Sirbucket) [reply docs/os/windows.md]: Saw what you said, a OK, I'll rework this though to remove the tutorial like format from youtube videos and make a simple google doc detailing optimal settings for performance, middle ground, and visuals

Comment (micwoj92) [reply docs/os/windows.md]: why google doc?

File: docs/os/windows.md (hunk 1 above)
Comment (Sirbucket) [reply docs/os/windows.md:67]: Is the third # supposed to mean it's a new section then?

File: docs/os/windows.md (hunk 2 above)
Comment (Sirbucket) [reply docs/os/windows.md]: It's simple to put it in there and put in a link, I can just figure out how to upload a text file link to github if that's what you want me to do?

Comment (Sirbucket) [reply docs/os/windows.md]: If you don't like the specific settings lists I can delete those and just have the details document

**Review by Sirbucket — COMMENTED** [review]
Also I do understand that both of those videos are pretty offensive so I completely get why you wouldn't want that in there anyways, I just liked them because the dude took time to look into the settings.

File: docs/os/windows.md (hunk 2 above)
Comment (micwoj92) [reply docs/os/windows.md]: Why do link to google docs when this is mastercomfig **docs**?

Comment (Sirbucket) [reply docs/os/windows.md]: I thought it'd be too cluttered, you'd need to add an entirely seperate section if you wanted me to include it in the mastercomfigs doc as a standalone.

Comment (Sirbucket) [reply docs/os/windows.md]: Well I mean I could do it myself but it'd be a stretch to add, easier to just shrink it down into 4 links

File: docs/os/windows.md (hunk 1 above)
Comment (mastercoms) [reply docs/os/windows.md:67]: Yes the third means it is a 3rd level header, thereby making a section under the 2nd level header above.

File: docs/os/windows.md [hunk 3]
```diff
@@ -79,10 +79,24 @@ Nahimic is an audio effects software preinstalled on some gaming laptops and des
...
 
 [Enabling Message Signaled Interrupts (PCIe MSI)](https://forums.guru3d.com/threads/windows-line-based-vs-message-signaled-based-interrupts-msi-tool.378044/) for all drivers is a great way to lower DPC latency caused by drivers (ring0). If you have an NVIDIA card, you can use [NVCleanstall](https://www.techpowerup.com/download/techpowerup-nvcleanstall/) to enable this, with the added benefit of fully customizing your driver install.
 
 Use [LatencyMon](https://www.resplendence.com/latencymon) and [DPC Latency Checker](https://www.thesycon.de/eng/latency_check.shtml) to analyze latency issues caused by kernel-mode device drivers. If they report issues, try updating your drivers or installing alternate drivers.
+
+## Nvidia/AMD Control Panel Settings
+
+Here will be listed a few documents detailing optimal settings.
+
+[DESCRIPTION](https://docs.google.com/document/d/1-k14uBdWv3RDIfdfvR6ICCA9wYgUYjflgd-NJkIFIiI/edit?usp=sharing): Describes each setting in a general overview
```
Comment (mastercoms) [inline docs/os/windows.md:94]: External links are not allowed in this fashion. Such a list of customizations should be tracked and reviewed in the docs.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/os/windows.md (hunk 3 above)
Comment (Sirbucket) [reply docs/os/windows.md:94]: Got it, I'll figure out a way to put it in without cluttering the page and I'll remove the change to the #

File: docs/os/windows.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ description: Learn how to optimize your Windows OS for better performance.
 
 # Windows
 
-## Enable Hardware-accelerated GPU scheduling
+## Enable Hardware-accelerated GPU Scheduling
```
Comment (Tiagoquix) [inline docs/os/windows.md:7]: Since you are capitalizing the titles, I think you should put the letter A from "accelerated" as a capital letter as well.

**Review by Tiagoquix — CHANGES_REQUESTED** [review]

**Comment by mastercoms** [comment]
Closing due to inactivity.

**Pull request closed without merging**